### PR TITLE
Add link for Creative Commons

### PR DESCRIPTION
**Pull request by HeySarahRay**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Review by amandeepmittal — APPROVED** [review]

**Review by nikrb — CHANGES_REQUESTED** [review]
link leads to 404 page

File: curriculum/LICENSE.md [hunk 1]
```diff
@@ -424,4 +424,4 @@ understandings, or agreements concerning use of licensed material. For
 the avoidance of doubt, this paragraph does not form part of the
 public licenses.
 
-Creative Commons may be contacted at creativecommons.org.
+Creative Commons may be contacted at [creativecommons.org.](creativecommons.org "Creative Commons")
```
Comment (nikrb) [inline curriculum/LICENSE.md]: markdown syntax is wrong and leads to a 404 page. I think the quoted words need removing in the round brackets

Comment (RandellDawson) [reply curriculum/LICENSE.md]: I am not sure where you are wanting the "Creative Commons" part to be, but if you are wanting it to appear to the write of `creativecommons.org.`, then "Creative Commons:" should be inside the `[ ]` and not the `( )`.  Also, you need a fully-formed url inside the `( )`.

[creativecommons.org "Create Commons"](https://creativecommons.org)

Comment (RandellDawson) [reply curriculum/LICENSE.md]: @ceejayoz An accident by me.

**Comment by RandellDawson** [comment]
Hi @blisscreates 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!
Happy Coding!

**Pull request closed without merging**